lib/aullar/styling.moon

```diff
@@ -170,6 +187,10 @@ define_class {
     ptr = @style_buffer\get_ptr(offset - 1, 1)
     style_map[ptr[0]]
 
+  get_nearest_style_marker: (pos) =>
```

Comment:
I think it may be better if the marker itself is not exposed via the API. Instead this call should just return the sub mode name directly. Also it could probably be renamed to `get_sub_mode_at(pos)` or something. Markers are an implementation detail here IMO.